### PR TITLE
Testing out nb pull requests

### PR DESCRIPTION
**Pull request by raspstephan**

### CODE REVIEW

File: devlog/12-probabilistic-dropout.ipynb [hunk 1]
```diff
@@ -36,6 +36,18 @@
     "This is my development branch of WeatherBench, which means things are not as nicely documented. Sorry about that. Regardless, let's import the modules we need. If you don't know what a function or class does, check out the definition in the `src` directory. Also check the requirements in the `environment.yml` file."
```
Comment (raspstephan) [inline devlog/12-probabilistic-dropout.ipynb:36]: <p>Test comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/raspstephan/WeatherBench/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: src/networks.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from tensorflow.keras.layers import *
 from tensorflow.keras import regularizers
 import numpy as np
-
+#tf.enable_eager_execution() #added. so as to be able to use numpy arrays easily
```
Comment (raspstephan) [inline src/networks.py]: Another test comment
